src/vm/dllimport.cpp

```diff
@@ -2256,6 +2256,11 @@ void NDirectStubLinker::DoNDirect(ILCodeStream *pcsEmit, DWORD dwStubFlags, Meth
                 // for managed-to-unmanaged CALLI that requires marshaling, the target is passed
                 // as the secret argument to the stub by GenericPInvokeCalliHelper (asmhelpers.asm)
                 EmitLoadStubContext(pcsEmit, dwStubFlags);
+#if defined(_TARGET_AMD64_) || defined(_TARGET_ARM64_)
+                // the secret arg has been shifted to left and ORed with 1 (see code:GenericPInvokeCalliHelper)
```

Comment:
Tag the other places we have found that depend on this with `see code:GenericPInvokeCalliHelper` so that they can be discovered later if somebody wants to come back to this?